docs/source-2.0/guides/building-models/build-config.rst

```diff
@@ -1665,6 +1719,115 @@ sources plugin. Lines that start with a number sign (#) are comments and are
 ignored. A Smithy manifest file is stored in a JAR as ``META-INF/smithy/manifest``.
 All model files referenced by the manifest are relative to ``META-INF/smithy/``.
 
+
+.. _run-plugin:
+
+run plugin
+----------
+
+The ``run`` plugin runs an external program during the build. This plugin is
+useful when integrating Smithy's build process with Smithy implementations
+that are not written in Java.
+
+When invoking the process, the Smithy model of the projection is serialized
+using the :ref:`JSON AST <json-ast>` and sent to the standard input of the
+process.
+
+.. important::
+
+    The ``run`` plugin requires a custom artifact name in its
+    :ref:`plugin ID <plugin-id>` (e.g., ``run::artifact-name``).
+
+The ``run`` plugin supports the following properties:
+
+.. list-table::
+    :header-rows: 1
+    :widths: 10 20 70
+
+    * - Property
+      - Type
+      - Description
+    * - command
+      - ``[string]``
+      - **REQUIRED** The name of the program to run, followed by an optional
+        list of arguments. If the command uses a relative path, Smithy will
+        first check if the command can be found relative to the current working
+        directory. Otherwise, the program must be available on the ``$PATH`` or
```

Comment:
I think it will be worth adding a note along the lines of 
> ``$PATH`` (if using a Unix-like environment or its equivalent for other operating systems)